scripts/rabbit/setupRabbit.py

```diff
@@ -165,7 +166,7 @@ def make_parser(parser=None):
         type=str,
         nargs="*",
         help="Don't run over processes belonging to these groups (only accepts exact group names)",
-        default=["QCD"],
+        default=["QCD", "WtoNMu_5", "WtoNMu_10", "WtoNMu_50"],
```

Comment:
this isn't very sustainable, e.g. if we run more signal masses, might make sense to define a sort of sample default list per analysis. Doesn't need to be in this PR

Reply:
Agreed but it doesn't really harm at this point and I don't have a good idea. Maybe later